base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -252,7 +252,7 @@ pub enum OutputManagerResponse {
     TransactionToSend(SenderTransactionProtocol),
     TransactionCancelled,
     SpentOutputs(Vec<UnblindedOutput>),
-    UnspentOutputs(Vec<UnblindedOutput>),
+    UnspentOutputs(Vec<(UnblindedOutput, u64)>),
```

Comment:
Rather create a strongly typed struct so that we know what this value is.

e.g. `Vec<UnspentUtxoInfo>`